packages/fx-core/src/plugins/resource/apiconnector/result.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+"use strict";
+import { FxError, SystemError, UserError, Result, ok } from "@microsoft/teamsfx-api";
```

Comment:
Just a reminder. The error construct has changed in hotfix branch.
We should update this after the dev merged hotfix branch.

Reply:
thanks